docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This dependency visibility ensures that the build system can accurately collect, inquire, analyze, and manage all dependencies during the build process.
+
+In {ProductName}, hermetic builds are achieved by blocking network access to the build process. This means that all the dependencies must be pre-fetched before the build can start.
+
+IMPORTANT: If you do not vendor all the required dependencies in a git repository before blocking network access, your build may fail. 
+
+Hermetic builds also focus on establishing a build pipeline that captures and maintains a precise record of the build's origin, dependencies, and modifications. For more information on the importance of provenance, see link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/concepts/slsa/con_slsa-conformity/[Supply chain security through SLSA conformity].
+
+
+.Prerequisites
+
+* You have an link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_upgrade_build_pipeline/[upgraded build pipeline].
+
+.Procedure
+
+For each component that you want to build hermetically:
+
+. Navigate to your component’s source code.
+
+. Open the *.tekton* directory and locate the *.yaml* files related to the *pull-request* and *push* processes.
+
+. *Configure the hermetic pipeline*: In both the *.yaml* files, add the following hermetic pipeline parameters.
+
++
+[source,yaml]
+----
+param:
```

Comment:
```suggestion
params:
```
In my pipeline definitions, it's params, not param.

Reply:
Should they be modifying this section? or the default value earlier in the definition? If the params: section, they would be replacing `value: $(params.hermetic)` with true, it's already there, just defaults to false.

Reply:
I think it makes sense to change the spec.params as that is easier to find and all parameters are colocated at the top.

Reply:
@arewm So I have to update `params` to `spec.params`, right?